code/modules/aberrants/_modification.dm

```diff
@@ -6,26 +6,29 @@
 	var/install_start_action = "attaching"
 	var/install_success_action = "attached"
 	var/install_time = WORKTIME_FAST
-	//var/install_tool_quality = null				
+	//var/install_tool_quality = null
 	var/install_difficulty = FAILCHANCE_ZERO
 	var/install_stat = STAT_COG
+	var/install_stat_robotic = STAT_COG //Used for robotic organs like FPB bones
```

Comment:
```suggestion
	var/install_stat_robotic = STAT_MEC //Used for robotic organs like FPB bones
```
I'd be reaaaaallly more happy with this

Reply:
I really just dont understand it what world it makes sense that a scientist would be better suited to this task then robotics?